.gitignore

```diff
@@ -40,7 +40,8 @@ npm-debug.log
 # Visual Studio Code
 .settings/launch.json
 .vs
-.vscode
+.vscode/*
+!.vscode/settings.json
```

Comment:
I suspect we should write `!/.vscode/settings.json`, otherwise this actually allows people to commit settings at _any_ level.